src/commands/registries/azure/deployImageToAzure.ts

```diff
@@ -141,3 +148,68 @@ class DockerSiteCreateStep extends AzureWizardExecuteStep<IAppServiceWizardConte
         return !context.site;
     }
 }
+
+class DockerWebhookCreateStep extends AzureWizardExecuteStep<IAppServiceWizardContext> {
+    public priority: number = 141; // execute after DockerSiteCreate
+
+    private _treeItem: RemoteTagTreeItemBase;
+
+    public constructor(treeItem: RemoteTagTreeItemBase) {
+        super();
+        this._treeItem = treeItem;
+    }
+
+    public async execute(context: IAppServiceWizardContext, progress: Progress<{ message?: string; increment?: number }>): Promise<void> {
+        const creatingNewApp: string = `Creating webhook for webapp "${context.newSiteName}"...`;
+        ext.outputChannel.appendLine(creatingNewApp);
+        progress.report({ message: creatingNewApp });
+
+        const site: Site = nonNullProp(context, 'site');
+        let siteClient = new SiteClient(site, context);
+        let appUri: string = (await siteClient.getWebAppPublishCredential()).scmUri;
+        if (this._treeItem instanceof AzureTagTreeItem) {
+            await this.createWebhookForApp(this._treeItem, context, appUri);
```

Comment:
nit: I'd put a notification once it's created here rather than putting it in the function itself.  That's just personal preference, but I like being able to control the notification where the ui is and keep it separate it from the implementation code.